src/views/Dashboard/Customer/Home/RequestTest/PaymentDetailCard.vue

```diff
@@ -239,34 +240,27 @@ export default {
     if (this.dataService.serviceFlow === "StakingRequestService") {
       this.stakingFlow = true
       const debioBalance = await getDbioBalance()
-
-      const stakingAmount = Number(this.stakingData.staking_amount.replaceAll(",", "")) * debioBalance.dbioToDai
-
-      this.stakingAmount = Number(formatPrice(stakingAmount)).toFixed(3)
-      const remainingStaking = this.dataService.price - stakingAmount
-      this.remainingDai = remainingStaking
-      this.remainingStaking = Number(formatPrice(remainingStaking)).toFixed(3)
-      this.remainingDbio = Number(formatPrice(remainingStaking / debioBalance.dbioToDai)).toFixed(3)
-
-      const excessAmount = stakingAmount - this.dataService.price
-      this.excessAmount = Number(formatPrice(excessAmount)).toFixed(3)
+      const stakingAmount = Number(formatPrice(this.stakingData.staking_amount))
+      this.stakingAmount = (stakingAmount * debioBalance.dbioToDai).toFixed(3)
+      this.remainingDai = this.dataService.totalPrice - this.stakingAmount
```

Comment:
better to rename the variable to `remainingAmount` to generalize, just in case we're not using DAI